sdk/documentintelligence/azure-ai-documentintelligence/samples/aio/sample_analyze_invoices_from_bytes_source_async.py

```diff
@@ -0,0 +1,247 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_analyze_invoices_from_bytes_source_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to analyze invoices.
+
+    See fields found on a invoice here:
+    https://aka.ms/azsdk/documentintelligence/invoicefieldschema
+
+USAGE:
+    python sample_analyze_invoices_from_bytes_source_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) DOCUMENTINTELLIGENCE_ENDPOINT - the endpoint to your Document Intelligence resource.
+    2) DOCUMENTINTELLIGENCE_API_KEY - your Document Intelligence API key.
+"""
+
+import os
+import asyncio
+
+
+async def analyze_invoice():
+    path_to_sample_documents = os.path.abspath(
+        os.path.join(
+            os.path.abspath(__file__),
+            "..",
+            "..",
+            "./sample_forms/forms/sample_invoice.jpg",
+        )
+    )
+
+    from azure.core.credentials import AzureKeyCredential
+    from azure.ai.documentintelligence.aio import DocumentIntelligenceClient
+    from azure.ai.documentintelligence.models import AnalyzeResult, AnalyzeDocumentRequest
+
+    endpoint = os.environ["DOCUMENTINTELLIGENCE_ENDPOINT"]
+    key = os.environ["DOCUMENTINTELLIGENCE_API_KEY"]
+
+    document_intelligence_client = DocumentIntelligenceClient(endpoint=endpoint, credential=AzureKeyCredential(key))
+    async with document_intelligence_client:
+        with open(path_to_sample_documents, "rb") as f:
+            # It's not expected to pass base64 bytes as the data will be encoded into base64 bytes in our SDK.
```

Comment:
Do we still need to specify this since we've renamed the property and the descriptions?

Reply:
We don;t need this line anymore, forgot to remove it. Thanks for catching!